src/components/Chat/Chat.vue

```diff
@@ -74,11 +74,21 @@
         <chat-toolbar :partner-id="partnerId">
           <template #avatar-toolbar>
             <ChatAvatar
+              v-if="enabledNodes"
               class="chat-avatar"
               :user-id="partnerId"
               use-public-key
               @click="onClickAvatar(partnerId)"
             />
+            <v-progress-circular
+              v-else
+              :class="`mr-3`"
+              v-show="!enabledNodes"
+              indeterminate
+              color="secondary"
+              size="30"
```

Comment:
Better to pass as a `number`

```suggestion
              :size="30"
```

Reply:
All the numbers (paddings, margins, width, height) in design system must be dibisible by 4 or 8. Better to use `32`.